llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -36,6 +36,11 @@ struct RISCVSupportedExtension {
   }
 };
 
+struct RISCVProfile {
```

Comment:
Very minor, but I believe you can use std::pair here instead.

Reply:
Very minor too, if you use std::pair, could you add the comments describing what the fields represent?